openapi.yaml

```diff
@@ -1,190 +1,513 @@
+      tags: 
+        - parks
+      description: Update park by unique ID
+      operationId: updateParkById
+      parameters:
+        - $ref: '#/parameters/parkIdPath'
+        - $ref: '#/parameters/parkPatchBody'
       responses:
         '200':
-          description: Successful response
+          description: Successful Response
           schema:
-            $ref: '#/definitions/PetsResult'
+            $ref: '#/definitions/ParkResult'
         '400':
-          description: Bad request
+          $ref: '#/responses/400'
+        '404':
+          $ref: '#/responses/404'
+        '409':
+          description: Resource conflict
           schema:
             $ref: '#/definitions/ErrorResult'
         '500':
-          description: Internal server error
+          $ref: '#/responses/500'
+    delete:
+      summary: Delete a specific park
+      tags: 
+        - parks
+      description: Delete park by unique ID
+      operationId: deleteParkById
+      parameters:
+        - $ref: '#/parameters/parkIdPath'
+      responses:
+        '204':
+          description: Park successfully deleted
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
+  /owners:
+    get:
+      summary: Lists all park owners
+      tags:
+        - owners
+      description: Get all park owners
+      operationId: getOwners
+      responses:
+        '200':
+          description: Successful Response
+          schema:
+            $ref: '#/definitions/OwnersResult'
+        '500':
+          $ref: '#/responses/500'
+    post:
+      summary: Creates a new park owner account
+      tags:
+        - owners
+      description: Post a new park owner
+      operationId: postOwner
+      parameters:
+      - $ref: '#/parameters/ownerPostBody'
+      responses:
+        '201':
+          $ref: '#/responses/201'
+        '400':
+          $ref: '#/responses/400'
+        '409':
+          description: Owner account already exists
           schema:
             $ref: '#/definitions/ErrorResult'
-  /pets/{id}:
+        '500':
+          $ref: '#/responses/500'
+  /owners/{ownerId}:
     get:
-      summary: Info for a specific pet
+      summary: gets the account of a park owner by ID
       tags:
-        - pets
-      description: Get pet by unique ID
-      operationId: getPetById
+        - owners
+      description: gets park owner by ID
+      operationId: getOwnerById
       parameters:
-        - $ref: '#/parameters/authorization'
-        - name: id
-          in: path
-          description: Unique Pet ID
-          required: true
-          type: string
+      - $ref: '#/parameters/ownerIdPath'
       responses:
         '200':
-          description: Successful response
+          description: Successful Response
           schema:
-            $ref: '#/definitions/PetResult'
+            $ref: '#/definitions/OwnerResult'
+        '400':
+          $ref: '#/responses/400'
```

Comment:
I was thinking it would be returned if invalid query parameters are provided, but after looking into it I found that the more common practice is to just ignore it. 

> In most cases, JSON API requires the server to return an error when it encounters an invalid value for a JSON API–defined query parameter. However, for API-specific query parameters (i.e. those not defined by JSON API), a server may choose to ignore an invalid parameter and have the request succeed, rather than respond with an error.